unstable/wlr-layer-shell-unstable-v1.xml

```diff
@@ -127,14 +127,19 @@
 
     <request name="set_exclusive_zone">
       <description summary="configures the exclusive geometry of this surface">
-        Requests that the compositor avoids occluding an area of the surface
-        with other surfaces. The compositor's use of this information is
```

Comment:
This could be interpreted as meaning an exclusive zone larger than the surface should be ignored. If this is intentional, it should be explicitly specified. I'm assuming that is not the meaning.